app/controllers/api/v1/conversations_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ConversationsController < ApiController
+      def index
+        @conversations = Conversation.where('sender_id = ? OR receiver_id = ?', conversation_user_id,
+                                            conversation_user_id)
```

Comment:
```suggestion
        @conversations = Conversation.where('sender_id = :user_id OR receiver_id = :user_id', user_id: current_user.id)
```
Take a look at [this](https://guides.rubyonrails.org/active_record_querying.html#placeholder-conditions)

Reply:
Also, this could be a scope at the model or a query object